src/pages/index.js

```diff
@@ -47,6 +47,43 @@ const featureRows = [
   ],
 ]
 
+const useCases = [
+  {
+    title: 'Your Friendly Name for the Web',
+    imageUrl: 'img/expert_light.svg',
+    imageUrlDark: 'img/expert_dark.svg',
+    link: 'https://w3n.id',
+    description: (
+      <>
+        Claim your web3name! A unique, human readable name that can be used as an alias for your DID.
+      </>
+    ),
+  },
+  {
+    title: 'Send Funds to Your DID',
+    imageUrl: 'img/tools.svg',
+    imageUrlDark: 'img/tools.svg',
+    link: 'docs/develop/sdk/quickstart',
+    description: (
+      <>
+        No need to remember long cryptic account addresses.
+        Add a service endpoint to your DID that specifies where asset should be send to.
```

Comment:
Can we change this? Since I am not sure if the audience would know what a service endpoint is without describing it to them. 

```suggestion
        Define where your assets are sent with your DID by using service endpoints.
        Time to explore what a DID can do for your convenience.
```